vllm/engine/async_llm_engine.py

```diff
@@ -411,11 +411,15 @@ async def add_request_async(
         lora_request: Optional[LoRARequest] = None,
         trace_headers: Optional[Mapping[str, str]] = None,
         prompt_adapter_request: Optional[PromptAdapterRequest] = None,
+        priority: int = 0,
     ) -> None:
         """Async version of :meth:`add_request`."""
         if lora_request is not None and not self.lora_config:
             raise ValueError(f"Got lora_request {lora_request} but LoRA is "
                              "not enabled!")
+        if priority > 0 and not self.scheduler_config.policy == "priority":
```

Comment:
```suggestion
        if priority and not self.scheduler_config.policy == "priority":
```
Negative is also allowed

Reply:
@schoennenbeck I see that you copied this from LLMEngine ... perhaps you could fix it there too?

Reply:
Sure, will do.

Reply:
You can also merge from `main` to fix the test failures now.

Reply:
Done and done. I also added the error handling to `add_request` in `AsyncLLMEngine` since it was previously missing.

Reply:
@DarkLight1337 The tests still seem to fail for reasons not related to the PR itself.